ABOUT.md

```diff
@@ -1,9 +1,106 @@
+
+#### Shogun is state-of-the-art
+
+* *efficient and competitive* implementation (from standard to cutting edge algorithms), modern software architecture in C++
+* easy combination of *multiple data representations, algorithm classes and general purpose tools* for rapid prototyping of data pipelines
+
+#### Shogun is open source
+
+* *free* and *community-based* machine learning education (see [Using Shogun to Learn](#useshoguntolearn))
+* *GPLv3 License* and working towards BSD compatibility
+
+You can try Shogun in our [cloud](https://cloud.shogun.ml) (no installation required) or find out more about our [mission](#mission), [features](#features) and [history](#shogunhistory).
+
+
+
+
+
+
+## <a name="mission"></a> Mission
+
+Machine learning is powerful. 
+Within the last two decades, machine learning (ML) has emerged from an eccentric academic field to transforming the way we live our everyday life. 
+Its methods influence society and its tools have a billion-dollar impact on global economy. 
+We want to make sure that the underlying machine learning algorithms are transparent and accessible. 
+We believe that ML should not be a privilege of corporations or political power with the money to buy technical knowledge. 
+Anyone - scientists, data enthusiasts, non-profit organisations, hackers, citizens, journalists, students - should be able to use ML tools for free.
+
+The Shogun ML toolbox promotes this mission through its features (*accessible*, *open source* and *good*) and a focus on ML education and developing its contributors.
+
+We are proud to be a big and very active open source ML platform.
+You can find the statistics of Shogun's development activity on [ohloh](https://www.openhub.net/p/shogun).
+
+## <a name="features"></a> Shogun features - *accessible*, *open source* and *good*
+
+### Accessible
+
+ * Shogun is **not limited to a single language**: 
+ you can use the toolbox through a unified interface from C++, Python, Octave, R, Java, Lua, C#, Ruby, etc. See [examples](http://shogun.ml/cookbook/latest/). 
+ Thus Shogun is independent of trends in computing languages and importantly, this allows you to use Shogun as a vehicle to expose your algorithm to multiple communities.  
+ * We use [SWIG](http://www.swig.org/) to enable *bidirectional* communication between C++ and all target languages.
+ * Shogun **runs natively under Linux/Unix, MacOS, and Windows**.
+ * Anyone can **try Shogun with minimal effort, for free**: 
+ simply by connecting to a [jupyter](http://jupyter.org/) notebook, users can run a docker instance on the Shogun [cloud](https://cloud.shogun.ml/hub/home)
+ (We offer this service for free through external donations).
+ * We offer as many binary installation packages as possible.
+ You can also just download our Docker image and run the Shogun cloud locally.
```

Comment:
put the link to dockerhub in the "docker" word `[Docker](https://hub.docker.com/r/shogun/)`